R/auto.R

```diff
@@ -93,27 +94,26 @@ pre_release_impl <- function(which, force) {
 }
 
 get_branch_name <- function() {
-  git2r::repository_head()$name
+  gert::git_branch()
 }
 
 get_remote_name <- function(branch) {
-  local <- git2r::branches(flags = "local")[[branch]]
-  upstream <- git2r::branch_get_upstream(local)
-  git2r::branch_remote_name(upstream)
+  branch_info <- gert::git_branch_list()
+  branch_info[branch_info$name == branch, ]$upstream
```

Comment:
The following code will only subset one column:

```suggestion
  # branch_info$name is unique: remote branches are prefixed with their remote name.
  branch_info$upstream[branch_info$name == branch]
```

I prefer this notation over the otherwise equivalent `df[row, "col"]` notation.